utils/build-presets.ini

```diff
@@ -722,6 +724,7 @@ llbuild
 swiftpm
 xctest
 foundation
+lbidispatch
```

Comment:
Yep totally is, will have that fixed tomorrow when I am at a machine to be able to edit it.

Sent from my iPhone

> On Aug 14, 2016, at 2:19 PM, Ryan Lovelett <notifications@github.com> wrote:
> 
> In utils/build-presets.ini:
> 
> > @@ -722,6 +724,7 @@ llbuild
> >  swiftpm
> >  xctest
> >  foundation
> > +lbidispatch
> I cannot help but feel this is a typo.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>